### ISSUE TITLE
sendNotificationMessage Hook ignored

### ISSUE THREAD

**Issue by janoschbatschi** [isotope/core#2174]
It seems that the Hook "sendNotificationMessage" is ignored if PayPal is selected as a payment module. When, e.g., Cash is selected the Hook gets called.
Contao: 4.4.54
Isotope: 2.6.14

**Comment by aschempp** [comment]
there is no `sendNotificationMessage` hook in Isotope eCommerce?

**Comment by janoschbatschi** [comment]
No, that's right. But due to the Hook being ignored only by the PayPal implementation which is part of Isotope eCommerce I thought this might be the right place. Using other payment methods the Hook gets called.

**Comment by aschempp** [comment]
did you use PayPal Standard or PayPal PLUS?

**Comment by janoschbatschi** [comment]
PayPal Standard

**Comment by aschempp** [comment]
this should be fixed by https://github.com/isotope/core/pull/2120
